kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -569,4 +577,26 @@ private String table(TableId table) {
   private Object lock(ConcurrentMap<TableId, Object> locks, TableId table) {
     return locks.computeIfAbsent(table, t -> new Object());
   }
+
+  private synchronized Map getCache() {
+    if (cache == null) {
+      cache = new HashMap<>();
+    }
+
+    return cache;
+  }
+
+  private Table retrieveCachedTable(TableId tableId) {
+    String key = tableId.getProject() + "." + tableId.getDataset() + "." + tableId.getTable();
+    Map<String, Table> cache = getCache();
+    Table table = cache.get(key);
+
+    if (table == null){
+      table = bigQuery.getTable(tableId);
+      if (table != null) cache.put(key, table);
+    }
+
+    return table;
+  }
+
```

Comment:
Looks like this section isn't used anymore; we should revert this and all the other changes on this file if they're not needed.

Reply:
File has been reverted to commit 26ecf5e9f8d3d524cea1dec999d10a222f74539f